### PR TITLE
Update Velero to v1.2.0

### PR DESCRIPTION
**Pull request by youssefazrak**
**What this PR does / why we need it**:
Update Velero to v1.2.0.
Minor upgrade. New features and bug fixes.

**Special notes for your reviewer**:
New mandatory flag `--plugins` to execute the `velero install` command that generates the CRDs.

**Documentation**:

Release note: https://github.com/vmware-tanzu/velero/releases/tag/v1.2.0

**Does this PR introduce a user-facing change?**:

```release-note
Update Velero to v1.2.0
```

### CODE REVIEW

**Comment by youssefazrak** [comment]
@xrstf Christoph, I updated the "update.sh" file with the `--plugins` flag as it is now a mandatory flag with Velero 1.2.0. When I launch the script with bash the formating is wrong (same in version 1.1.0) so that might be due to a different output with a newer version of Velero. It seems the update.sh was written for Velero 1.0.0.
Also, do you know if this is a manually triggered file or is it triggered by ci when a cluster is deployed? I can't seem to find any mention of "update.sh".
Cheers!

**Comment by youssefazrak** [comment]
@xrstf I changed the update.sh script to work with the latest version of yq (v3.1.1). Now I have a proper output.
Regarding the newly generated CRD.yaml file, I guess we can remove the new annotation `controller-gen.kubebuilder.io/version: (unknown)`?
Also I suppose we need to apply those manually? 
Cheers

File: config/backup/velero/values.yaml [hunk 1]
```diff
@@ -3,8 +3,8 @@ velero:
   # if you are using restic, make sure to use an official image
   # that also contains the restic binary
   image:
-    repository: gcr.io/heptio-images/velero
-    tag: v1.1.0
+    repository: velero/velero
```
Comment (xrstf) [inline config/backup/velero/values.yaml]: Please always use a fully-qualified Docker image name, i.e. `docker.io/velero/velero`.

**Comment by xrstf** [comment]
> Regarding the newly generated CRD.yaml file, I guess we can remove the new annotation controller-gen.kubebuilder.io/version: (unknown)?

Nah, why remove them? They offer no real value, but they also don't hurt.

> Also I suppose we need to apply those manually?

Helm applies the CRDs for us, like with everything in `templates/`.

**Review by xrstf — CHANGES_REQUESTED** [review]
minor

File: config/backup/velero/values.yaml (hunk 1 above)
Comment (youssefazrak) [reply config/backup/velero/values.yaml]: Updated

**Pull request merged**